resources/views/components/menu/account.blade.php

```diff
@@ -7,24 +7,21 @@
 $user = request()->user();
 ?>
 @guest
-    <ul class="navbar-nav">
-        @if($settings->get('auth.registration', true))
-            <li class="nav-item hidden lg:inline-block">
-                {{--<a class="nav-link" href="{{ route('register') }}">--}}
-                <a class="nav-link" href="{{ url('createaccount.php') }}">
-                    <span class="sr-only">{{ __('Register') }}</span>
-                    <span class="hidden md:inline-block">{{ __('Register') }}</span>
-                </a>
-            </li>
-        @endif
-        {{--<li class="nav-item">
-            <a class="nav-link" href="{{ route('login') }}">
-                <x-fas-power-off />
-                <span class="sr-only">{{ __('Sign In') }}</span>
-                <span class="hidden lg:inline-block">{{ __('Sign In') }}</span>
+    <div class="nav-item">
+        <a class="nav-link" href="{{ route('login') }}">
+            <x-fas-power-off class="mr-1" />
```

Comment:
It feels odd for Sign In to have an icon, but not Sign Up.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/ff74f906-8cb4-4cc8-9eb0-e33f50ded990)

Perhaps `fas-user-plus` for sign up, and `fas-user-pen` for sign in? 
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/0fcb7949-613d-4c71-8673-b3c07cb35a76)

Reply:
Was thinking about that, too, but got close to removing the icon from "Sign in" rather
(I used the very same on completionist.me 😅 which is fine)
That looks very bare on the other hand 
<img width="355" alt="Screenshot 2023-09-05 at 21 13 26" src="https://github.com/RetroAchievements/RAWeb/assets/1280590/f43e48bc-4d13-40fd-9631-b62103d81871">

Reply:
Updated in https://github.com/RetroAchievements/RAWeb/pull/1831/commits/0d90580495763a63bf61ef6ac7b376d928eec545
![Screenshot 2023-09-06 at 23 07 55](https://github.com/RetroAchievements/RAWeb/assets/1280590/9b4a774d-6f3c-4845-94de-2fe356360dc7)
![Screenshot 2023-09-06 at 23 08 06](https://github.com/RetroAchievements/RAWeb/assets/1280590/c4d68d87-a985-4edb-ab98-b3fe3ed60553)